solara/components/echarts.vue

```diff
@@ -14,7 +14,24 @@ module.exports = {
       this.echarts = echarts;
       this.create();
     })();
-  },
+    console.log("responsive:"+str(this.responsive));
+    if(this.responsive){
+    this.resizeObserver = new ResizeObserver(entries => {
+    for (let entry of entries) {
+      if (entry.target === this.$refs.echarts) {
+        this.handleResize();
+      }
+    }
+  });
+  this.resizeObserver.observe(this.$refs.echarts);
+};
+},
+beforeDestroy() {
+  if (this.resizeObserver) {
+    this.resizeObserver.unobserve(this.$refs.echarts);
+    this.resizeObserver.disconnect();
+  }
+},
```

Comment:
Please remove the log and format/indent the code similar to the rest of the document style.